test/integration/infra.go

```diff
@@ -134,19 +134,20 @@ func (infra *infra) setup() error {
 	if err != nil {
 		return err
 	}
-
+	debug := true
```

Comment:
Add this to `infra` and expose as a flag in driver.go?

Reply:
EnableCoreDump, Version, MeshConfigMapName don't seem to have flags ? is it really necessary for this one?
but let me look

Reply:
`DebugMode` is something people might want to set for tests. `Version` and `MeshConfigMapName` shouldn't need to changed for e2e tests.

Reply:
fixed